Backend/Models/Database/Pharmacy.cs

```diff
@@ -26,6 +26,8 @@ public class Pharmacy
 
         public ICollection<PharmacyWorkingHours> PharmacyWorkingHours { get; set; }
 
+        public ICollection<User> Pharmacists { get; set; }
+
```

Comment:
Maybe it would be more correct to just use `Employess` here? Not every employee working at a pharmacy could be a pharmacist (for example, some sort of manager).

Reply:
Yes, but according to our UML class diagram only Pharmacists have a direct connection to a Pharmacy. Other types of workers are unrelated.

Reply:
What if in the future we added some new role, for example PharmacyAdmin? They would not be pharmacists but still would work in the pharmacy so it would be logic to retrieve them along with other users. Your approach and UML is not future-proof.

Reply:
When we add those roles we will change it but for now it would cause confusion to a newcomer as he/she would assume that every type of employee has direct access to a pharmacy which is not true (suppliers, manufacturers and others do not have that). So either we leave it as Pharmacists, change UML logic or go for a stricter name other than Employees.